FirebaseAnalyticsSwift/README.md

```diff
@@ -43,7 +43,77 @@ struct ContentView: View {
   }
 }
 ```
+An example that demonstrates how the custom event logging API and manual screen view event logging API can make the code more efficient and reduce the number of lines required for event logging.
 
+### Before (Without APIs)
+
+```swift
+struct ContentView: View {
+    var body: some View {
+        VStack {
+            Text("Welcome to our App!")
+                .padding()
+            Button("Click Me!") {
+                // Logging a custom event when the button is clicked.
+                Analytics.logEvent("button_clicked", parameters: nil)
+            }
+        }
+        .onAppear {
+            // Logging the screen view event when the ContentView appears.
+            Analytics.logEvent(AnalyticsEventScreenView, parameters: [AnalyticsParameterScreenName: "main_content"])
+        }
+    }
+}
+```
+
+### After (With APIs)
+
+```swift
+struct ContentView: View {
+    var body: some View {
+        VStack {
+            Text("Welcome to our App!")
+                .padding()
+            Button("Click Me!") {
+                // Using the new custom event logging API to log the button click.
+                Analytics.logCustomEvent(name: "button_clicked")
+            }
+        }
+        // Using the new manual screen view event logging API to log the screen view.
+        .analyticsScreen(name: "main_content")
+    }
+}
+
+// Introducing a custom event logging API.
+extension Analytics {
+    static func logCustomEvent(name: String, parameters: [String: Any]? = nil) {
+        Analytics.logEvent(name, parameters: parameters)
+    }
+}
```

Comment:
Does this wrapper provide any additional utility over the method it's wrapping?

Reply:
We appreciate the suggestion, but this repo should focus on documenting the Firebase APIs as they are defined. 

The APIs have already been through a review process and it's fine for any particular app to wrap as they would like.

Please remove the `logCustomEvent` wrapper.

Reply:
ok
right on it

Reply:
I am removing the whole custom event logging API

Reply:
take a look now, if i have to make any changes

Reply:
@paulb777 sir can we connect on Linkedin? Stange place to ask but i don't know how else to reach you